src/cli/commands/create-config.ts

```diff
@@ -0,0 +1,61 @@
+import {CliCommand} from "./interface";
+import {CommanderStatic} from "commander";
+import logger from "../../logger";
+import fs from "fs";
+import {stringify} from "@iarna/toml";
+import defaults from "../../node/defaults";
+import {CliError} from "../error";
+
+interface ICreateConfigOptions {
+  outputFile: string;
+}
+
+export class CreateConfigCommand implements CliCommand {
+  public register(commander: CommanderStatic): void {
+    commander
+      .command("create-config")
+      .description("Create default config file")
+      .option("-o, --outputFile [output_file]", "Path to output file destination")
+      .action(async (options) => {
+        // library is not awaiting this method so don't allow error propagation 
+        // (unhandled promise rejections)
+        try {
+          await this.action(options);
+        } catch (e) {
+          logger.error(e.message);
+        }
+      });
+  }
+
+  public async action(options: ICreateConfigOptions): Promise<void> {
+    if (options.outputFile) {
+      if (fs.existsSync(options.outputFile)){
+        throw new CliError(`${options.outputFile} already exists`);
+      }
+
+      // Stringify defaults into TOML format
+      const defaultsToWrite = (({chain, db, rpc, eth1}) => ({
```

Comment:
Config generation could also go in config util class. Generally these commander methods are awkward to test so we should keep them as small as possible

Reply:
could also just add a method to the `CreateConfigClass`, not sure how useful it will be to have a seperate file for.